src/ValidationContext.php

```diff
@@ -248,11 +248,7 @@ public function getAttributeLabel(): ?string
      */
     public function getTranslatedAttribute(): ?string
     {
-        if ($this->attribute === null) {
-            return null;
-        }
-
-        $label = $this->attributeLabel ?? $this->attribute;
+        $label = $this->attributeLabel ?? $this->attribute ?? 'The value';
```

Comment:
May be use `Value` without `The`? 
It's will be more consistent with attribute names, for example `age`.

Also, suggest for attributes up first symbold, for exmaple `age` → `Age`.

It would be ideal to be able to use several tokens:
• `{attribute}` — `age` or `value`
• `{Attribute}` — `Age` or `Value`

It allow to create pretty message.